pkgs/development/python-modules/cairocffi/default.nix

```diff
@@ -0,0 +1,60 @@
+{ buildPythonPackage
+, fetchPypi
+, lib
+, substituteAll
+, makeFontsConf
+, freefont_ttf
+, pytest
+, glibcLocales
+, cairo
+, cffi
+, withXcffib ? false, xcffib
+, python
+, glib
+, gdk_pixbuf }:
+
+buildPythonPackage rec {
+  pname = "cairocffi";
+  version = "0.8.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0i9m3p39g9wrkpjvpawch2qmnmm3cnim7niz3nmmbcp2hrkixwk5";
+  };
+
+  LC_ALL = "en_US.UTF-8";
+
+  # checkPhase require at least one 'normal' font and one 'monospace',
+  # otherwise glyph tests fails
+  FONTCONFIG_FILE = makeFontsConf {
+    fontDirectories = [ freefont_ttf ];
+  };
+
+  checkInputs = [ pytest glibcLocales ];
+  propagatedBuildInputs = [ cairo cffi ] ++ lib.optional withXcffib xcffib;
+
+  checkPhase = ''
+    py.test $out/${python.sitePackages}
+  '';
+
+  # FIXME: make gdk_pixbuf dependency optional
+  # Happens with 0.7.1 and 0.7.2
+  # OSError: dlopen() failed to load a library: gdk_pixbuf-2.0 / gdk_pixbuf-2.0-0
```

Comment:
Have you tested whether this is still a problem?

Reply:
Maybe @avnik and @FRidh, who wrote these comments, can tell us more.

Reply:
Not really sure how to reproduce that.

Reply:
Thinking about it, this probably talks about the reason for patch, not an another thing to fix. I should move the `FIXME` comment elsewhere, to make it less confusing.

Reply:
And ignore drop the version info, it will happen as long as they do not use absolute paths.